dev/ci/docker_linux/Dockerfile

```diff
@@ -98,33 +98,11 @@ RUN touch ~/.android/repositories.cfg
 
 # Add npm to path.
 ENV PATH="/usr/bin:${PATH}"
-RUN dpkg-query -L nodejs
-# Install Firebase
-# This is why we need nodejs installed.
-RUN /usr/bin/npm --verbose install -g firebase-tools@10.1.2
-
-# Install golang more recent than in repo (necessary for dashing to build)
-RUN curl 'https://dl.google.com/go/go1.13.3.linux-amd64.tar.gz' | tar xvz
-ENV GOBINARY=/go/bin/go
```

Comment:
I'm confused, I thought the docs-linux shard depended on this.

HOWEVER, on pre-submit, the docs-linux shard did not trigger a rebuild of the docker image: https://cirrus-ci.com/task/5053363346735104

The change to the Dockerfile here should have invalidated the cache. Or am I missing something?

Reply:
The way I thought this worked was that, prior to a linux build, cirrus would hash the dockerfile (the path is specified here: https://github.com/flutter/flutter/blob/master/.cirrus.yml#L20), and if that hash was already cached with a pre-built image, that image would be used. I would think that this change would invalidate that cache.

Reply:
The image was built in the previous commit [d013c07](https://github.com/flutter/flutter/pull/97556/checks?check_run_id=5015601055), the last commit didn't trigger an image build because it only changed a README file.

Reply:
Ahh, I see.